Registry/Registry.EM_COMMON

```diff
@@ -3086,6 +3086,8 @@ package   drip           sf_surf_irr_alloc==2        -             state:irrigat
 package   sprinkler      sf_surf_irr_alloc==3        -             state:irrigation,irr_rand_field
 
 
+package   lesscheme      bl_pbl_physics==0           -             -
+package   nopblscheme    bl_pbl_physics==0           -             -
```

Comment:
you have two names for this?

Reply:
@dudhia 
Jimy,
I can remove one. Wei asked about if we should call it nopbl or les. We could call it both. What is your preference?

Reply:
OK, LESscheme it is!